specification/authorization/resource-manager/Microsoft.Authorization/preview/2018-05-01-preview/authorization-AccessReviewCalls.json

```diff
@@ -1026,6 +1026,474 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.Authorization/roleAssignmentApprovals": {
```

Comment:
>roleAssignmentApprovals [](start = 40, length = 23)

You are modifying a stable API version, which is not allowed as per guidance at [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning). You should add these changes in a new API version.

Reply:
Actually this API version was introduced very recently - last month(in this PR : https://github.com/Azure/azure-rest-api-specs/pull/10445). Is it okay to add these APIs in this version(since the version is very new) or do we always need to add new API versions for any chance?

Reply:
Also, this is not a stable version but a preview version

Reply:
I've moved new APIs to a different version : 2021-01-01-preview. Thanks for bringing up this point!